script/server/index.ts

```diff
@@ -6,6 +6,8 @@ import serveStatic from 'serve-static';
 import config from '../../src/config';
 import setupProxy from '../../src/setupProxy';
 
+export { Server } from 'http';
```

Comment:
whats this for?

Reply:
Probably a leftover from when I was using the proxy. I'll take it out.